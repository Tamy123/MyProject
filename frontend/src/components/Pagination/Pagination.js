import React from "react";
import "./Pagination.css"

const Pagination = props => {
  return (
    <React.Fragment>
      {props.mapper.map((item, i) => {
        return (
          <a
            key={i}
            href="#"
            onClick={props.handlePagination}
            className={i + 1 === Number(props.currentPage) ? "active" : ""}
          >
            {i + 1}
          </a>
        );
      })}
    </React.Fragment>
  );
};

export default Pagination;
