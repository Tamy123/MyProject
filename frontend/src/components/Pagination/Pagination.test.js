import React from "react";
import { shallow } from "enzyme";
import Pagination from "./Pagination";

describe("sample text ", () => {
  let props;
  let getShallowComponent;
  let wrapper;

  beforeEach(() => {
    props = {
      mapper: [{}, {}]
    };
    getShallowComponent = () => shallow(<Pagination {...props} />);
    wrapper = getShallowComponent();
  });
  describe("sample text ", () => {
    test("render", () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
