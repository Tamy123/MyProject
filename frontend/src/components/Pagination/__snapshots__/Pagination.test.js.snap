// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample text  sample text  render 1`] = `ShallowWrapper {}`;
